pandas/tools/tests/test_util.py

```diff
@@ -291,6 +291,79 @@ def test_non_hashable(self):
         with self.assertRaisesRegexp(TypeError, "Invalid object type"):
             pd.to_numeric(s)
 
+    def test_downcast(self):
+        mixed_data = ['1', 2, 3]
+        int_data = [1, 2, 3]
+        date_data = np.array(['1970-01-02', '1970-01-03',
+                              '1970-01-04'], dtype='datetime64[D]')
+
+        invalid_downcast = 'unsigned-integer'
+        msg = 'invalid downcasting method provided'
+
```

Comment:
I suppose, but I mention it because it isn't a requirement before merging a PR.  `numpy` has an Appveyor check for PR's, but then again, their test suite is MUCH, MUCH faster to run than the one for `pandas`.  It's amazing how much slower that suite becomes when run on Appveyor.